main/level_1.gd

```diff
@@ -33,9 +35,16 @@ func _on_enemy_spawner_timer_timeout() -> void:
 	else:
 		random_y = randf_range(spawn_area_y, -spawn_area_y)
 		random_x = possible_spawn_x.pick_random()
+	
+	if random_x == spawn_area_x:
+		random_x += -27
+	elif random_y == spawn_area_y:
+		random_y += -27
 
 	enemy_instance.position = Vector2(random_x, random_y)
 	add_child(enemy_instance)
+	
+	enemy_instance.initialize_enemy(player)
```

Comment:
https://docs.godotengine.org/en/stable/classes/class_object.html#class-object-method-call-deferred

Let's try to keep consistency, below we are creating new bullets with:`new_bullet`

```suggestion
	new_enemy.call_deferred("initialize_enemy", player)
```

What do you think? :)